src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,125 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+//import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage {
+
+    String newNameTest = "TestNewPlaylist";
+    String renameTest = "Renamed Playlist";
+
+    @FindBy(css = "[img.avatar]")
+    WebElement userAvatarIcon;
+    @FindBy(css = ".playlist:nth-child(3)")
+    WebElement playlistInput;
+    @FindBy(css = "i[data-testid='sidebar-create-playlist-btn']")
+    WebElement playlistNewBtn;
+    @FindBy(xpath = "//li[@data-testid='playlist-context-menu-create-simple']")
+    WebElement playlistContextMenu;
+    @FindBy(css = "[name='name']")
+    WebElement playlistInputField;
+    @FindBy(xpath = "//div[@class='song-list-controls']//button[@class='del btn-delete-playlist']")
+    WebElement contextDeletePlaylist;
+    @FindBy(xpath = "//div[@class='alertify-logs top right']/div[@class='success show'][1]")
+    WebElement playlistCreatedMsgLocator;
+    @FindBy(xpath = "//*[@id='playlists']/ul/li[3]/a")
+    WebElement clickPlaylistLocator;
+    @FindBy(xpath = "//div[@class='song-list-controls']//button[@class='del btn-delete-playlist']")
+    WebElement clickDeletePlaylistLocator;
+    @FindBy(css = ".playlist:nth-child(3)")
+    WebElement doubleClickPlaylistName;
+    @FindBy(xpath = "//div[@class='alertify-logs top right']/div[@class='success show'][2]")
+    WebElement msgDeletedPlaylistLocator;
+    @FindBy(css = "input[name='name']")
+    WebElement enterPlaylistNameLocator;
+    @FindBy(xpath = "//a[text()='\" + renameTest + \"']")
+   WebElement confirmNewPlaylistLocator;
+    @FindBy(xpath = "//*[@id='playlistWrapper']/div//div[1]/table//td[2]")
+    WebElement firstSongInThePlaylist;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```